hypopt/model_selection.py

```diff
@@ -240,35 +246,35 @@ def fit(
 
         X_val : np.array of shape (n0, m)
             The validation data to optimize paramters with. If you do not provide this,
-            cross validation on the training set will be used. 
+            cross validation on the training set will be used.
```

Comment:
Most of your line changes aren't line changes?

Reply:
Yeah. I think my IDE reformatted some of these lines. I have no idea why this happened.